@navikt/core/icons/svg/ErrorFilled.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12 0C18.6274 0 24 5.37258 24 12C24 18.6274 18.6274 24 12 24C5.37258 24 0 18.6274 0 12C0 5.37258 5.37258 0 12 0ZM15.5714 7L17 8.42857L13.428 12L17 15.5714L15.5714 17L12 13.428L8.42857 17L7 15.5714L10.572 12L7 8.42857L8.42857 7L12 10.572L15.5714 7Z" fill="#262626"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 24C18.6274 24 24 18.6274 24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 18.6274 5.37258 24 12 24ZM16.5714 18L18 16.5714L7.42857 6L6 7.42857L16.5714 18Z" fill="#262626"/>
```

Comment:
Jeg har aldri sett et sånt error-ikon før. Burde vi ikke bruke et uttrykk for å kommunisere feil som folk kjenner fra før?

Reply:
Designerne har funnet frem til denne nå etter flere møter slik jeg har skjønt det. Men er med på å gi litt pushback da jeg heller ikke kjenner til dette utrykket for error.